src/DevBetterWeb.Web/appsettings.Local.json

```diff
@@ -0,0 +1,90 @@
+{
+	"googleReCaptcha:SiteKey": "6LcEwN4ZAAAAAC-_UfJ0dmdU8bRxLLJ4FwjXyE1R",
+	"googleReCaptcha:SecretKey": "YOUR_SECRET_KEY",
+	"storageconnectionstring": "[connection string goes here]",
+	"videoUrlPrefix": "https://devbetter2storage.blob.core.windows.net/videos/",
+	"GoogleMapsAPIKey": "GOOGLE_MAPS_API_KEY",
+	"ConnectionStrings": {
+		"DefaultConnection": "Server=database,1433;MultipleActiveResultSets=true;User Id=sa;Password=bigStr0ngPwd*;Encrypt=false"
```

Comment:
Are these secrets safe to store in source control? Why store appsettings.Local.json in source control at all? Wouldn't it be better to ignore this file and then you wouldn't have to worry about accidently checking in secrets when you update it locally.